pyuvdata/uvflag/uvflag.py

```diff
@@ -1862,6 +2318,23 @@ def __add__(
             )
             this.Npols = len(this.polarization_array)
 
+        if axis in ["baseline", "antenna", "time"]:
```

Comment:
I think I'm not seeing something. If we're adding two objects over time, why do we need to concatenate their antenna_numbers, names, and positions? shouldn't those be unchanged. Or are they shaped as Nblts and I'm imagining them as Nants_data?

Reply:
So a couple lines down after these concatenations, I drop down to the unique set. This allows us to combine objects which do not have exactly matching antenna numbers, names and positions. But maybe they should be required to be matching, which would fit with the compatibility checking above. Also, as written, if there were different names or positions for the same antenna number this would silently take the values from the first object, which is not ideal.

Reply:
I made this better, it now only does it if really required and warns the user it's happening.